### PR TITLE
feat(collector): apply recent job link and finished date

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- apply recent job link and finished date
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 1]
```diff
@@ -225,9 +225,15 @@ const state = reactive({
 
             const userCurrentTime = dayjs.tz(current, storeState.timezone);
             const hours = props.item.schedule.hours ?? [];
-            const nextScheduledHour = hours.sort((a, b) => a - b).find((num) => num > userCurrentTime.hour());
-
-            const nextScheduledTime = current.set('h', nextScheduledHour || 0).set('m', 0);
+            const sortedHours = hours.sort((a, b) => a - b);
+            const nextScheduledHour = sortedHours.find((num) => num > userCurrentTime.hour());
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:233]: praise: far better!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 2]
```diff
@@ -225,9 +225,15 @@ const state = reactive({
...
+
+            let nextScheduledTime;
+            if (nextScheduledHour) {
+                nextScheduledTime = current.set('h', nextScheduledHour || 0).set('m', 0);
+            } else {
+                nextScheduledTime = current.add(1, 'day').set('h', sortedHours[0]).set('m', 0);
+            }
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:240]: praise: understandable!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 3]
```diff
@@ -81,7 +81,7 @@
                     >
                         <p-tooltip v-if="jobStatus.status === JOB_STATE.SUCCESS"
                                    class="icon-fill-wrapper success"
-                                   :contents="$t('INVENTORY.COLLECTOR.MAIN.JOB_SUCCESS', {date: 'yyyy-mm-dd hh:mm:ss'})"
+                                   :contents="$t('INVENTORY.COLLECTOR.MAIN.JOB_SUCCESS', {date: dayjs(jobStatus.finished_at).format('YYYY-MM-DD hh:mm:ss')})"
                                    position="top"
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:85]: issue: I think this must be converted to user timezone!

**Review by WANZARGEN — APPROVED** [review]
Clear code!
There seems to be an issue, so please check it :)

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue (hunk 3 above)
Comment (yuda110) [reply apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:85]: 🥸
dayjs.utc(jobStatus.finished_at).tz(store.state.user.timezone).format('YYYY-MM-DD hh:mm:ss')

**Review by yuda110 — APPROVED** [review]
LGTM~

**Pull request merged**